src/store/plugins/names.js

```diff
@@ -72,8 +76,12 @@ export default store =>
                 { root: true },
               );
             }
+            if (name.expiresAt <= height && oldName.autoExtend) {
```

Comment:
```suggestion
            if (name.expiresAt >= height && oldName.autoExtend) {
```

This way we will ensure we dispatch updatePointer only in the case when the current block height has not reached the name's expiration date.